source/ship/shi_dragon/shi_dragon.ship

```diff
@@ -0,0 +1,399 @@
+addAbility(NewShipType, "HyperSpaceCommand", 1, 1, 20, 80, 0, 3)
+addAbility(NewShipType, "ParadeCommand", 1)
+addAbility(NewShipType, "WaypointMove")
+addAbility(NewShipType, "GuardCommand", 1, 3000, 500)
+addAbility(NewShipType, "HyperspaceViaGateCommand", 1, 3, .15, 0.3)
+addAbility(NewShipType, "RetireAbility", 1, 0)
+addAbility(NewShipType, "CanAttack", 1, 1, 0, 0, 0.35, 1,
+    "Fighter, Fighter_hw1, Corvette, Corvette_hw1, Frigate, Utility, Resource,,,, SmallCapitalShip, BigCapitalShip, Mothership",
+    "Fighter_vs_Frigate",
+    { Fighter = "Fighter_vs_Fighter" },
+    { Fighter_hw1 = "Fighter_vs_Fighter" },
+    { Corvette = "Fighter_vs_Corvette" },
+    { Corvette_hw1 = "Fighter_vs_Corvette" },
+    { Frigate = "Fighter_vs_Frigate" },
+    { SubSystem = "TopAttack_Interceptor_vs_Subsystem" },
+    { SmallCapitalShip = "Fighter_vs_CapShip" },
+    { BigCapitalShip = "Fighter_vs_CapShip" },
+    { Mothership = "Fighter_vs_Mothership" },
+    { ResourceLarge = "Fighter_vs_ResourceLarge" })
+
+addAbility(NewShipType, "CustomCommand", 1, "Speed Burst", 1, 0, 38, 0, 1, 0.1, 0,
+    "data:scripts/custom_scripts/Afterburner.lua",
+    "Start_Afterburner", "Do_Afterburner", "Finish_Afterburner", "shi_dragon", 0.5, 3, 1)
+
+--------------------------------------------------
+--- MODEL MUST LOAD AT THIS POINT
+LoadModel(NewShipType, 1)
+--------------------------------------------------
+
+--------------------------------------------------
+--- Weapons
+--------------------------------------------------
+
+StartShipWeaponConfig(NewShipType, "gun_shivheavylaser", "Weapon_Gun1", "")
+StartShipWeaponConfig(NewShipType, "gun_shivheavylaser", "Weapon_Gun2", "")
+StartShipWeaponConfig(NewShipType, "miss_rockeye", "Weapon_Missile", "")
+
+--------------------------------------------------
+--- HardPoints
+--------------------------------------------------
+
+--StartShipHardPointConfig(NewShipType,"Shield","Hardpoint_Shield","System","Innate","Damageable","shield_dragon","","","","","","","","")
+--StartShipHardPointConfig(NewShipType, "Gun_1", "Weapon_Gun1", "Weapon", "Production", "Destroyable", "wep_shiheavylaser", "wep_prometheusfs1", "", "", "", "", "", "", "")
+--StartShipHardPointConfig(NewShipType, "Gun_1", "Weapon_Gun2", "Weapon", "Production", "Destroyable", "wep_shiheavylaser", "wep_prometheusfs1", "", "", "", "", "", "", "")
+--StartShipHardPointConfig(NewShipType, "Missile1", "Weapon_Missile", "Weapon", "Sensors", "Destroyable", "wep_shiharpoon", "wep_rockeye", "wep_empadv", "wep_tempest", "wep_hornet", "wep_tornadonormal", "wep_harpoon", "wep_trebuchet", "")
+
+--------------------------------------------------
+--- Latch Points
+--------------------------------------------------
+
+
+
+--------------------------------------------------
+--- Addendum
+--------------------------------------------------
+
+loadShipPatchList(NewShipType, "data:sound/sfx/ship/", 0, "", "", 1, "ShivanFighter", "", 3, "", "Root")

```

Comment:
For these old FS1 ships, the sound effects file is different, should be `sf_old` instead of `ShivanFighter` in this case, you can look them all up in `sound/sfx/ship/`